README.md

```diff
@@ -49,7 +49,7 @@ You can add streams by setting an eventlistener on the MergeStream instance.
 To remove a stream, just end the stream itself.
 ```javascript
 connectionsReadStream.on("data", function (connection) {
-	connectionsReadStream.addConnectionsStream(['newStream'], newConnectionsReadStream]);
+	connectionsReadStream.addConnectionsStream(['newStream', newConnectionsReadStream]);
```

Comment:
I think we should rework this in a different pull request in which we have this instead:
```javascript
connectionsReadStream.addConnectionsStream('newStream', newConnectionsReadStream);
```
Makes more sense code-wise